packages/adapter-nuxt2/src/hooks/project-init.ts

```diff
@@ -47,11 +51,37 @@ const configurePrismicModule = async ({
 		}
 	}
 
-	const mod = await loadFile(nuxtConfigPath);
-	const config =
-		mod.exports.default.$type === "function-call"
-			? mod.exports.default.$args[0]
-			: mod.exports.default;
+	const mod = await loadFile(nuxtConfigPath); // here maybe
+
+	let config;
+	try {
+		config =
+			mod.exports.default.$type === "function-call"
+				? mod.exports.default.$args[0]
+				: mod.exports.default;
+	} catch {
+		const errorMessage = `Failed modify ${path.basename(nuxtConfigPath)}`;
```

Comment:
@samlfair Could you take a look at this sentence "Failed modify ./nuxt.config.js" and provide something else if needed?

Reply:
I think "Failed to update ..." is more common indeed

Reply:
yep

Reply:
done